.github/workflows/build.yml

```
@@ -0,0 +1,42 @@
+  build:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: 'Checkout code'
+        uses: actions/checkout@v5
+        with:
+          fetch-depth: 1
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: 'Set image tag and name'
+        id: tag
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2193:warning:2:27: The arguments to this comparison can never be equal. Make sure your syntax is correct

(shellcheck)